blueprints/acceptance-test/files/tests/acceptance/__name__-test.js

```diff
@@ -17,14 +17,14 @@ describe('Acceptance: <%= classifiedModuleName %>', function() {
   });
 
   afterEach(function() {
-    Ember.run(application, 'destroy');
+    application.destroy();
   });
 
-  it('can visit /<%= dasherizedModuleName %>', function() {
+  beforeEach(function() {
```

Comment:
I didn't find anything explicitly mentioning `moduleForAcceptance` in terms of a formal proposal over at `ember-test-helpers`. Maybe I'm missing something? In any case, I can certainly have a look.